go.mod

```diff
@@ -19,47 +19,22 @@ require (
 	golang.org/x/sys v0.13.0
 )
 
+replace github.com/Bedrock-OSS/regolith => ./regolith
```

Comment:
I'm not sure if I understand the reason behind adding this. On Linux it doesn't seem to do anything. On Windows it just prevents me from building the executable. Do you have some kind of special build workflow that requires that? The `replace` is typically used if you need to work on a dependency locally, but I don't think it makes much sense here since you're replacing THIS repository.

Btw. We had soo many unused modules here... it's good to run `go mod tidy` every now and then. Thanks!